lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -33,14 +33,21 @@ class HuskyProject():
     api_token_env_var: str
     command_name: str
 
-    def __init__(self, project_id: int, lang: str, api_token_env_var: str, command_name: str) -> None:
+    # The date represented by REDCap record repeat instance #1.
+    repeat_instance_1_date: datetime
```

Comment:
I'd expect to use a `date` instead of `datetime` object: https://docs.python.org/3.5/library/datetime.html#date-objects

Dates are much simpler than datetimes and would avoid lots of complex logic around timezones that we don't need or want in this case.

Reply:
We don't need this code anymore, so I'm removed it.